style.css

```diff
@@ -458,3 +457,151 @@
 .icon {
   margin-right: 20px;
 }
+
+.contact {
+  width: 100%;
+  background: #6070ff;
+  background-image: url(.Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: 143px 11px;
+  border-top-left-radius: 80px;
+  margin-top: -70px;
+  padding: 103px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 100px;
+  align-items: center;
+}
```

Comment:
- [ ] The background image should match the one on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (the one on figma has shapes yours doesn't), kindly change that: 😁

![image](https://user-images.githubusercontent.com/20567503/177543288-9604d776-8b92-46a3-9db1-46c1b95b5d37.png)